spec/values/ParticipleSpec.js

```diff
@@ -0,0 +1,36 @@
+var Participle = require( "../../js/values/Participle.js" );
+
+describe( "A test for checking the Participle", function() {
+	it( "checks the properties of a participle object without a passive", function() {
+		var mockParticiple = new Participle( "geschlossen", "Es wird geschlossen worden sein.", [ "wird", "worden" ],  "irregular" );
```

Comment:
What happens if you put a string and not an array in the Participle object for the Auxiliaries or even a number. How does the Participle react to that?